cmd/getStorageValue.go

```diff
@@ -0,0 +1,67 @@
+// Copyright © 2020 NAME HERE <EMAIL ADDRESS>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Should probably remove the Apache license

Reply:
🤦‍♀ I don't know how that keeps getting in there! My goland copyright profile is empty. 🤔 Thanks for catching that!